include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -423,7 +423,7 @@ class format_parse : public format_base
                 if ((*option_it)[id_size] == '=') // -key=value
                 {
                     if ((*option_it).size() == id_size + 1) // malformed because no value follows '-i='
-                        throw parser_invalid_argument("Value cast failed for option " +
+                        throw invalid_argument("Value cast failed for option " +
                                                       prepend_dash(id) +
```

Comment:
since the name changes, this line and the one below needs to be indented correctly
```cpp
throw invalid_argument("Value cast failed for option " +
                       prepend_dash(id) +
                       ": No value was provided.");
```
This could be the case for several changes.